rendered/traces-only/secret.yaml

```diff
@@ -0,0 +1,15 @@
+---
+# Source: splunk-otel-collector/templates/secret.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: splunk-otel-collector
+  labels:
+    app: splunk-otel-collector
+    chart: splunk-otel-collector-0.24.13
+    release: default
+    heritage: Helm
+type: Opaque
+data:
+  splunk_access_token: Q0hBTkdFTUU=
```

Comment:
I think we need to add at least a comment on how to get this string from your token. Another option would be to avoid creating secrets with a helm option.

Reply:
```
$ pbpaste | base64 --decode
CHANGEME
```
💯